lib/funding_credit.js

```diff
@@ -1,8 +1,16 @@
 'use strict'
 
+const numberValidator = require('./validators/number')
+const amountValidator = require('./validators/amount')
+const symbolValidator = require('./validators/symbol')
```

Comment:
Nice, I really like the separation of the validator functins

Reply:
Thanks :slightly_smiling_face: